spec/features/submissions_spec.rb

```diff
@@ -197,6 +197,14 @@
           expect(page).to have_css(".has-error")
         end
 
+        scenario "sortowanie zgłoszeń względem daty nadesłania" do
```

Comment:
Trzeba to przenieść do kontekstu `2 zgłoszenia w bazie danych` - obecnie jest w `brak autora w bazie danych`